VisualActivityViewController.swift

```diff
@@ -183,8 +183,10 @@ final class VisualActivityViewController: UIActivityViewController {
     override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
         
-        UIView.animate(withDuration: fadeOutDuration) {
-            self.preview?.alpha = 0
+        UIView.animate(withDuration: fadeOutDuration, animations: {
+            self.preview?.alpha = 0;
```

Comment:
```suggestion
            self.preview?.alpha = 0
```
One final thing, I missed this at first glance, can you remove the semi colon?

Reply:
Sorry, Objective C head in action there! Coming right up.

Reply:
No problem, I've been there as well.